### ISSUE TITLE
Issue in focus

### ISSUE THREAD

**Issue by jaivardhan-bhola** [martinet101/win32mica#7]
By clicking on the part where mica is used, the program minimizes. If the topmost attribute is applied, the window loses focus, and the mica effect is no longer visible.

**Comment by sumeshir26** [comment]
Thats a tkinter bug, nothing to do with win32mica

**Comment by martinet101** [comment]
> Thats a tkinter bug, nothing to do with win32mica
Yes, this might be true, since on Qt win32mica behaves as expected

**Comment by rdbende** [comment]
> Thats a tkinter bug, nothing to do with win32mica

It's not even a bug. It's a feature.

> You can try setting a semi-transparent (but not visible) bg color (sth like rgba(255, 255, 255, 1%))

Well, Tkinter doesn't know such thing as "semi-transparent"

**Comment by martinet101** [comment]
> > Thats a tkinter bug, nothing to do with win32mica
> 
> It's not even a bug. It's a feature.
> 
> > You can try setting a semi-transparent (but not visible) bg color (sth like rgba(255, 255, 255, 1%))
> 
> Well, Tkinter doesn't know such thing as "semi-transparent" 😕

One thing you could do is creating with photoshop a white image and set it to 1% opacity, and then set it as the background on your tkinter app

**Comment by rdbende** [comment]
I believe you can't do that either. In Tkinter every widget has a background box, that can't have any transparency, so even if you have an image with alpha channel, it will blend with the background field.
I remember when making my themes for Tkinter I suffered for a long time before I realized that it's simply not possible.
You can make windows with alpha channel, but then everything, including the window decorations becomes semi-transparent.
Transparency is just an area, where Tcl/Tk simply sucks.

**Comment by martinet101** [comment]
Well, then the best thing I can say is to migrate to another UI toolkit.

I can personally recommend you Qt, because it provides lots of flexibility, performance with high quality. 
I also reached a point where tkinter wasn't _enough_, and I decided to fully convert the project I was working on to Qt, and while it was a tedious process, I can guarantee that I have never regretted doing the change.
